### PR TITLE
CCD-3734 Update validation in Verify NoC Answers to include

### PR DESCRIPTION
**Pull request by padmapriyanalam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-3734

### Change description ###
Update validation in Verify NoC Answers to include 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/managecase/service/noc/ChallengeAnswerValidatorTest.java [hunk 1]
```diff
@@ -210,6 +210,275 @@ void shouldErrorWhenFieldIsNullButAnswerProvided() {
         assertThat(exception.getErrorMessage(), is(ANSWERS_NOT_MATCH_LITIGANT.getErrorMessage()));
     }
 
+    @Test
+    void whenCaseFieldIsPresentAndAnswerIsNull_IgnoreNullFieldIsTrue() {
```
Comment (danlysiak) [inline src/test/java/uk/gov/hmcts/reform/managecase/service/noc/ChallengeAnswerValidatorTest.java:214]: Could you double-check the names of the new tests please - they don't seem to quite match up correctly e.g. this one is an error scenario but it's the following test which mentions an error (but is actually a success!)

Comment (padmapriyanalam) [reply src/test/java/uk/gov/hmcts/reform/managecase/service/noc/ChallengeAnswerValidatorTest.java]: updated the test names to match with the functionality

**Review by danlysiak — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -379,7 +379,7 @@ dependencies {
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
 
-  testImplementation (group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.13.0') {
+  testImplementation (group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.7-prerelease') {
```
Comment (rajaram-nagarajan) [inline build.gradle:382]: we need to use the version 7.19.7

Comment (padmapriyanalam) [reply build.gradle]: changed the version to 7.19.7

**Pull request merged**